### ISSUE TITLE
Incompatibility with build 232.X and above

### ISSUE THREAD

**Issue by davidmszrs** [patrickfv/horizon-intellij#1]
Updating to IntelliJ IDEA 2023.2 automatically disables the theme and the following message is displayed while checking the plugin in the market place:

<img width="415" alt="image" src="https://github.com/patrickfv/horizon-intellij/assets/18365917/24469a13-f28b-4545-adf2-367031831faa">

Affected versions: build 232.X and above (IntelliJ based IDEs 2023.2 and above)
Steps to reproduce: upgrade to the latest version. The theme will automatically get disabled.

**Comment by elkhoudiry** [comment]
Yes, i love this theme and i would like to continue to use it on newer ide version
